gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/services/UserOrganizationService.java

```diff
@@ -54,7 +56,21 @@ public List<UserOrganization> getAll(UserOrganizationCriteria criteria, Pageable
 
     }
 
-    public void setAdminRoleForOrganizationsOwner(Organization organization) {
+    public UserOrganization createUserOrganization(UserOrganization userOrganization) throws Exception{
```

Comment:
It should check that the user-organization combination doesn't already exists.
In the flow it is used currently it is sure that will not exists. 

Since this is going to be a reusable function, in future scenarios, the combination might exist, so it should handle this by throwing an error